manifests/j/JanDeDobbeleer/OhMyPosh/3.162.0/JanDeDobbeleer.OhMyPosh.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.162.0
+PackageLocale: en-US
+Publisher: Jan De Dobbeleer
+PackageName: Oh My Posh

```

Comment:
```suggestion
PackageName: Oh my Posh
```

Reply:
![image](https://user-images.githubusercontent.com/11600822/120895538-4ccc4380-c615-11eb-8b84-251a88d870a8.png)

Reply:
@ItzLevvie wouldn't it make more sense to have the program renamed? Because everything else also references `Oh My posh`

Reply:
Since you're the developer, it's entirely up to you. I don't see any reason to change it because this is what shows up in Control Panel; and for `winget upgrade` to work it needs to match.

Reply:
I'm adjusting the name, as everything else seems to be with upper case. Thanks.